### ISSUE TITLE
Data Issue: Massachusetts no longer reporting daily county-level data?

### ISSUE THREAD

**Issue by harrislapiroff** [nytimes/covid-19-data#429]
### Describe the issue:
- [ ] Incorrect number of cases or deaths
- [ ] Suspicious number of cases or deaths
- [x] Missing data for a locality
- [ ] Missing time-series information
- [ ] Other

### Fuller details
Looks like Massachusetts hasn't reported county level data at all for a few days. Not sure if this is a permanent change. Might be worth a note.

**Comment by tiffehr** [comment]
Yes, Massachusetts has removed their daily county data.  But it continues to include it in a weekly report.  Our county data will now only update weekly, unless the state chooses to return to the daily reporting methodology.

We'll add a note once we're confident this is a permanent change.

cc/ @albertsun

**Comment by ghop02** [comment]
I just received word that they are resuming publishing confirmed cases. https://www.mass.gov/doc/covid-19-raw-data-august-19-2020/download in County.csv.

I'm not sure why they chose to just report confirmed cases, rather than confirmed + probable

**Comment by tiffehr** [comment]
Yep, we're aware.  Good to confirm.

**Comment by ghop02** [comment]
Great, thanks! Not sure if you had any conversation with them, but do you know why they decided to report only confirmed? I'm pretty sure this is different than they were reporting before.

Anyways, thanks for being on top of this!
